packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2118,6 +2135,9 @@ class TextInput {
           firstArg['action'] as String,
           firstArg['data'] == null ? <String, dynamic>{} : firstArg['data'] as Map<String, dynamic>,
         );
+      case 'TextInputClient.performCustomAction':
```

Comment:
Maybe this should go in services/binding.dart, where we handle onDismissSystemContextMenu.